Extensions/IISWebAppDeploy/Src/vss-extension.json

```diff
@@ -2,7 +2,7 @@
   "manifestVersion": 1,
   "extensionId": "iiswebapp",
   "name": "IIS Web App Deployment Using WinRM",
-  "version": "1.2.4",
+  "version": "1.2.5",
```

Comment:
IMO, re-usable modules is a big change. We can update Minor version. Though functionally not required - it will help us remember the changes when we are dealing with customer issues in future